qbt_migrate/classes.py

```diff
@@ -86,8 +90,9 @@ def discover_relevant_fast_resume(cls, bt_backup_path: str, existing_path: str,
     @classmethod
     def backup_folder(cls, folder_path: str, archive_path: str):
         cls.logger.info(f'Creating Archive {archive_path} ...')
+        files_to_archive = os.listdir(folder_path)
```

Comment:
This fix was not implemented in #18 merge. This stops the issue of archiving the archive file if the backup dir was set to the archive path.